package.json

```diff
@@ -9,60 +9,35 @@
   "main": "gulpfile.js",
   "devDependencies": {
-    "@automattic/calypso-build": "5.1.0",
-    "@babel/core": "7.8.7",
-    "@wordpress/babel-preset-default": "4.7.0",
-    "@wordpress/blocks": "6.8.0",
-    "@wordpress/components": "8.4.0",
-    "@wordpress/data": "4.10.0",
-    "@wordpress/editor": "9.8.0",
-    "@wordpress/i18n": "3.7.0",
-    "@wordpress/jest-preset-default": "1.0.6",
-    "@wordpress/scripts": "5.1.0",
-    "babel-eslint": "10.1.0",
-    "babel-loader": "8.0.6",
-    "browser-sync": "2.26.7",
-    "cross-env": "5.2.1",
-    "css-loader": "3.4.2",
-    "del": "3.0.0",
-    "dotenv": "8.2.0",
+    "@automattic/calypso-build": "6.0.0",
+    "@wordpress/babel-preset-default": "4.12.1",
+    "@wordpress/jest-preset-default": "6.0.0",
+    "@wordpress/scripts": "8.0.1",
+    "cross-env": "7.0.2",
+    "del": "5.1.0",
     "eslint": "6.8.0",
-    "eslint-config-prettier": "6.6.0",
-    "eslint-plugin-react": "7.18.3",
-    "eslint-wpcom": "npm:eslint@5.16.0",
-    "fs-extra": "7.0.1",
+    "eslint-config-prettier": "6.11.0",
     "gulp": "4.0.2",
-    "gulp-babel": "8.0.0",
     "gulp-checktextdomain": "2.2.1",
-    "gulp-chmod": "2.0.0",
-    "gulp-clean-css": "4.2.0",
-    "gulp-rename": "1.4.0",
-    "gulp-sass": "4.0.2",
     "gulp-sort": "2.0.0",
-    "gulp-uglify": "3.0.2",
-    "gulp-wp-pot": "2.4.0",
+    "gulp-wp-pot": "2.4.2",
     "gulp-zip": "4.2.0",
-    "jest": "25.0.0",
-    "mini-css-extract-plugin": "0.4.5",
-    "node-sass": "4.13.0",
-    "orchestrator": "0.3.8",
-    "sass-loader": "7.3.1",
-    "style-loader": "0.21.0",
-    "webpack": "4.41.2",
-    "webpack-cli": "3.3.10"
+    "prettier": "npm:wp-prettier@1.19.1"
   },
   "dependencies": {
     "select2": "4.0.5"
   },
   "scripts": {
     "test": "npm run test-php && npm run test-js",
     "test-php": "./vendor/bin/phpunit",
+    "build": "cross-env NODE_ENV=production npx calypso-build",
+    "start": "calypso-build --watch",
+    "package": "gulp package",
     "test-js": "wp-scripts test-unit-js --passWithNoTests",
-    "block-editor-assets": "cross-env NODE_ENV=production npx calypso-build --config webpack.block-editor-assets.config.js --env.WP",
-    "block-editor-assets:dev": "npx calypso-build --config webpack.block-editor-assets.config.js --env.WP",
-    "block-editor-assets:watch": "npx calypso-build --config webpack.block-editor-assets.config.js --env.WP --watch",
-    "lint-js": "./node_modules/eslint/bin/eslint.js assets/block-editor --ext=js,jsx",
-    "lint-js:fix": "./node_modules/eslint/bin/eslint.js assets/block-editor --ext=js,jsx --fix",
-    "lint-js-wpcom": "./node_modules/eslint-wpcom/bin/eslint.js --no-eslintrc --config .eslintrc.wpcom.json --ignore-path .eslintignore-wpcom ."
+    "lint-css": "wp-scripts lint-style **/*.scss",
+    "lint-css:fix": "npm run lint-css -- --fix",
+    "run-phpcs": "./vendor/bin/phpcs --encoding=utf-8",
+    "lint-js": "wp-scripts lint-js",
+    "lint-js:fix": "npm run lint-jsx -- --fix"
```

Comment:
We have to update the script here now to `npm run lint-js`

Reply:
Always missing this :) dbf50a5